docs/proposals/disable-fencing.txt

```diff
@@ -0,0 +1,31 @@
+Disable Machine Fencing 
```

Comment:
why would you need this at all since mhc targets only machines of your choice via label selector?

Reply:
We have a maintenace controller, that should drain all VM's from the respective node, and also we do not want to fence it, because of it under the maintenance, but its temporary state and we do not want each time update node or MHC labels, because it easy to forgot to do it and disable fencing for the machine at all

Reply:
isn't this proposing exactly to use a label?

Reply:
this one who will set the label is node-maintenance controller, we just want respect this label under MHC and do not fence node
Added under the document

Reply:
I think this feature is useful. In some cases, it maybe desired to skip specific machines.
Another approach could be to support list of labels, that guide nodes to be skipped, as a configuration option on mhc